### PR TITLE
Pioneer 17 CSM deploy csv updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
UPDATE: 2021-11-05 - all SM deployment/recovery info updated.  ISSM-13 will not be recovered on P17 Leg 1.
See attached QLR tables for reference.  
[Mooring Tables_3204-01702_Quick_Look_Cruise_Report_Coastal_Pioneer_17_Leg_1_2021-11-05_DE.pdf](https://github.com/WHOIGit/ooicgsn-asset-management/files/7488240/Mooring.Tables_3204-01702_Quick_Look_Cruise_Report_Coastal_Pioneer_17_Leg_1_2021-11-05_DE.pdf)

Preliminary updates based on info in #cg-data-team channel; still pending deployment dates/times, recovery dates/times

### CODE REVIEW

File: deployment/CP01CNSM_Deploy.csv [hunk 1]
```diff
@@ -441,71 +441,71 @@ AR48,,AR52,,CP01CNSM-SBD12-04-PCO2AA000,13,1,2020-10-29T14:57:00,2021-04-03T12:0
...
+AR61,,,,CP01CNSM-SBD11-01-MOPAK0000,15,1,2021-11-03T14:45:00,,CGMCP-01CNSM-00015,,CGINS-MOPAK0-11841,40.139823,-70.771173,,0,132,,,
+AR61,,,,CP01CNSM-SBD11-02-HYDGN0000,15,1,2021-11-03T14:45:00,,CGMCP-01CNSM-00015,,CGINS-HYDGN0-11407,40.139823,-70.771173,,0,132,,,
+AR61,,,,CP01CNSM-SBD11-06-METBKA000,15,1,2021-11-03T14:45:00,,CGMCP-01CNSM-00015,,CGINS-METLGR-00007,40.139823,-70.771173,,0,132,,,
+AR61,,,,CP01CNSM-SBD12-00-DCLENG000,15,1,2021-11-03T14:45:00,,CGMCP-01CNSM-00015,,CGCON-BDCL12-00009,40.139823,-70.771173,,0,132,,,
+AR61,,,,CP01CNSM-SBD12-03-HYDGN0000,15,1,2021-11-03T14:45:00,,CGMCP-01CNSM-00015,,CGINS-HYDGN0-49860,40.139823,-70.771173,,0,132,,,
+AR61,,,,CP01CNSM-SBD12-04-PCO2AA000,15,1,2021-11-03T14:45:00,,CGMCP-01CNSM-00015,,CGINS-PCO2AA-33177,40.139823,-70.771173,,0,132,,,
+AR61,,,,CP01CNSM-SBD12-05-WAVSSA000,15,1,2021-11-03T14:45:00,,CGMCP-01CNSM-00015,,CGINS-WAVSSA-04571,40.139823,-70.771173,,0,132,,,
+AR61,,,,CP01CNSM-SBD12-06-METBKA000,15,1,2021-11-03T14:45:00,,CGMCP-01CNSM-00015,,CGINS-METLGR-00003,40.139823,-70.771173,,0,132,,,
+AR61,,,,CP01CNSM-SBD12-08-FDCHPA000,15,1,2021-11-03T14:45:00,,CGMCP-01CNSM-00015,,CGINS-FDCHPB-03904,40.139823,-70.771173,,-5,132,,,
```
Comment (reedan88) [inline deployment/CP01CNSM_Deploy.csv:511]: Looks good.

File: deployment/CP03ISSM_Deploy.csv [hunk 2]
```diff
@@ -403,34 +403,34 @@ AR52,,,,CP03ISSM-SBD11-06-METBKA000,13,1,2021-04-02T20:47:00,,CGMCP-03ISSM-00013
...
+AR61,,,,CP03ISSM-RID27-04-DOSTAD000,14,1,2021-11-01T14:10:00,,CGMCP-03ISSM-00014,,CGINS-DOSTAD-00410,40.3646,-70.888252,,7,93,,,
+AR61,,,,CP03ISSM-SBC11-00-CPMENG000,14,1,2021-11-01T14:10:00,,CGMCP-03ISSM-00014,,CGCON-BCPM01-50004,40.3646,-70.888252,,0,93,,,
+AR61,,,,CP03ISSM-SBD11-00-DCLENG000,14,1,2021-11-01T14:10:00,,CGMCP-03ISSM-00014,,CGCON-BDCL11-50004,40.3646,-70.888252,,0,93,,,
+AR61,,,,CP03ISSM-SBD11-01-MOPAK0000,14,1,2021-11-01T14:10:00,,CGMCP-03ISSM-00014,,CGINS-MOPAK0-11839,40.3646,-70.888252,,0,93,,,
+AR61,,,,CP03ISSM-SBD11-02-HYDGN0000,14,1,2021-11-01T14:10:00,,CGMCP-03ISSM-00014,,CGINS-HYDGN0-41692,40.3646,-70.888252,,0,93,,,
+AR61,,,,CP03ISSM-SBD11-06-METBKA000,14,1,2021-11-01T14:10:00,,CGMCP-03ISSM-00014,,CGINS-METLGR-00033,40.3646,-70.888252,,0,93,,,
+AR61,,,,CP03ISSM-SBD12-00-DCLENG000,14,1,2021-11-01T14:10:00,,CGMCP-03ISSM-00014,,CGCON-BDCL12-50004,40.3646,-70.888252,,0,93,,,
+AR61,,,,CP03ISSM-SBD12-03-HYDGN0000,14,1,2021-11-01T14:10:00,,CGMCP-03ISSM-00014,,CGINS-HYDGN0-49868,40.3646,-70.888252,,0,93,,,
+AR61,,,,CP03ISSM-SBD12-04-PCO2AA000,14,1,2021-11-01T14:10:00,,CGMCP-03ISSM-00014,,CGINS-PCO2AA-33182,40.3646,-70.888252,,0,93,,,
```
Comment (reedan88) [inline deployment/CP03ISSM_Deploy.csv:436]: Looks good.

File: deployment/CP04OSSM_Deploy.csv [hunk 3]
```diff
@@ -340,98 +340,98 @@ AR39,,AR48,,CP04OSSM-SBD11-06-METBKA000,11,1,2019-09-29T14:46:00,2020-11-08T13:1
...
+AR61,,,,CP04OSSM-RID27-04-DOSTAD000,14,1,2021-10-30T20:10:00,,CGMCP-04OSSM-00014,,CGINS-DOSTAD-00126,39.940462,-70.877457,,7,445,,,
+AR61,,,,CP04OSSM-SBC11-00-CPMENG000,14,1,2021-10-30T20:10:00,,CGMCP-04OSSM-00014,,CGCON-BCPM01-50002,39.940462,-70.877457,,0,445,,,
+AR61,,,,CP04OSSM-SBD11-00-DCLENG000,14,1,2021-10-30T20:10:00,,CGMCP-04OSSM-00014,,CGCON-BDCL11-50002,39.940462,-70.877457,,0,445,,,
+AR61,,,,CP04OSSM-SBD11-01-MOPAK0000,14,1,2021-10-30T20:10:00,,CGMCP-04OSSM-00014,,CGINS-MOPAK0-44005,39.940462,-70.877457,,0,445,,,
+AR61,,,,CP04OSSM-SBD11-02-HYDGN0000,14,1,2021-10-30T20:10:00,,CGMCP-04OSSM-00014,,CGINS-HYDGN0-Z5974,39.940462,-70.877457,,0,445,,,
+AR61,,,,CP04OSSM-SBD11-06-METBKA000,14,1,2021-10-30T20:10:00,,CGMCP-04OSSM-00014,,CGINS-METLGR-00038,39.940462,-70.877457,,0,445,,,
+AR61,,,,CP04OSSM-SBD12-00-DCLENG000,14,1,2021-10-30T20:10:00,,CGMCP-04OSSM-00014,,CGCON-BDCL12-50002,39.940462,-70.877457,,0,445,,,
+AR61,,,,CP04OSSM-SBD12-03-HYDGN0000,14,1,2021-10-30T20:10:00,,CGMCP-04OSSM-00014,,CGINS-HYDGN0-Z5975,39.940462,-70.877457,,0,445,,,
+AR61,,,,CP04OSSM-SBD12-04-PCO2AA000,14,1,2021-10-30T20:10:00,,CGMCP-04OSSM-00014,,CGINS-PCO2AA-33183,39.940462,-70.877457,,0,445,,,
```
Comment (reedan88) [inline deployment/CP04OSSM_Deploy.csv:437]: Looks good.

**Review by reedan88 — COMMENTED** [review]
Looks good.

**Review by reedan88 — APPROVED** [review]

**Comment by rtravis-whoi** [comment]
[Pioneer 17 AM Updates [DT-151]](https://trello.com/c/LYsz2Yeu/151-pioneer-17-am-updates-dt-151)

File: deployment/CP01CNSM_Deploy.csv (hunk 1 above)
Comment (cdobs) [inline deployment/CP01CNSM_Deploy.csv:511]: Looks good

File: deployment/CP03ISSM_Deploy.csv (hunk 2 above)
Comment (cdobs) [inline deployment/CP03ISSM_Deploy.csv:436]: Looks good

File: deployment/CP04OSSM_Deploy.csv (hunk 3 above)
Comment (cdobs) [inline deployment/CP04OSSM_Deploy.csv:437]: Looks good

**Review by cdobs — APPROVED** [review]
Looks good

**Pull request merged**